passive/Server Header Disclosure.js

```diff
@@ -14,10 +15,38 @@ function scan(ps, msg, src)
 
     var url = msg.getRequestHeader().getURI().toString();
     var headers = msg.getResponseHeader().getHeaders("Server")
+    var headers_string = headers.toString();
```

Comment:
This will fail if the `Server` header is not present.

Reply:
added this inside function after checking for header != null